### PR TITLE
added option to toggle button

### PR DESCRIPTION
**Pull request by caseyyee**
Use by adding hideButton option when initializing new instance of WebVRManager:
`var vrmanager = new WebVRManager(effect, { hideButton: true })`

### CODE REVIEW

**Comment by caseyyee** [comment]
Use by adding hideButton option when initializing new instance of WebVRManager:
`var vrmanager = new WebVRManager(effect, { hideButton: true })`

File: js/webvr-manager.js [hunk 1]
```diff
@@ -63,6 +68,11 @@ var Modes = {
   IMMERSED: 3,
 };
 
+var ButtonVisibility = {
```
Comment (borismus) [inline js/webvr-manager.js]: Do we need this structure? Isn't a boolean enough?

File: js/webvr-manager.js [hunk 2]
```diff
@@ -109,18 +119,26 @@ WebVRManager.prototype.createVRButton = function() {
   return button;
 };
 
+WebVRManager.prototype.displayVRButton = function(value) {
```
Comment (borismus) [inline js/webvr-manager.js]: How about setVisibility(bool isVisible)?

**Comment by caseyyee** [comment]
Ok made adjustments as per comments!

**Comment by borismus** [comment]
Thanks!!

**Pull request merged**